front/src/components/chips/CompanyChip.tsx

```diff
@@ -4,6 +4,7 @@ import styled from '@emotion/styled';
 export type CompanyChipPropsType = {
   name: string;
   picture?: string;
+  clickHandler?: (editing: boolean) => void;
```

Comment:
by naming convention, props should be called "onClick". We do that for all event handler in props

Reply:
but I actually think that we should expose a onRemoveClick rather than onClick

Reply:
onRemoveClick is a good name... I really didn't want to call it onClick because it didn't get an event arg 😅

Reply:
onRemoveClick?: () => void;